src/seer_utils.erl

```diff
@@ -0,0 +1,72 @@
+-module(seer_utils).
+
+-include("seer.hrl").
+
+-export([carbon_format/1]).
+
+carbon_format(Metrics) ->
+    Timestamp = erlang:timestamp(),
```

Comment:
If you use `integer_to_binary(erlang:system_time(second))`, you can remove `unix_time_binary/1`